README.md

```diff
@@ -290,6 +302,19 @@ Manual DNS configuration can be specified via a list of addresses
  properties, respectively. If specified, it determines the route metric
 for DHCP assigned routes and the default route, and thus the priority for multiple interfaces.
 
+Static route configuration can be specified via a list of routes given in the `route`
+option. The default value is an empty list. Each route is a dictionary with the following
+entries: `network`, `prefix`, `gateway` and `metric`. `network` and `prefix` together specify
+the destination network. CIDR notation or network mask notation are not supported yet. If the
+boolean option `route_append_only` is `True`, the specified routes are appended to the
+existing routes, if it is `False` (default), the current routes are replaced. Setting this
+option without setting `route` has the effect of preserving the current static routes. The
+boolean option `rule_append_only` works in a similar way for routing rules. Note that there is
+no further support for routing rules at the moment, so this option serves merely the purpose
+of preserving the current routing rules.  Note also that when
+`route_append_only`/`rule_append_only` is not specified, the current routes/routing rules will
+be deleted by the role.
```

Comment:
Yes, the later line clarifies on that... but it's a bit duplicated, isn't it?

Reply:
It is, but I find the current behavior (remove all existing routes) dangerous, so a bit of redundancy does not harm.